main.js

```diff
@@ -0,0 +1,15 @@
+const feed = document.getElementById('feed')
+
+fetch('https://newsdata.io/api/1/news?apikey=pub_37020dad134a463f0666e9f1a5eba6e8b2a34&q=coding%20%20AND%20computer&language=en&category=technology')
```

Comment:
Do you think there is any need to hide this API key?

Reply:
I'm sorry, you mean we need to hide it? It was a public key given by the website so I didn't think too much about it